src/app/admin/AdminPage.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import React, { ReactElement } from "react";
+import UsersTable from "@/app/admin/usersTable/UsersTable";
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import { ExpandMore } from "@mui/icons-material";
+import TableSurface from "@/components/Tables/TableSurface";
+import CreateUserForm from "@/app/admin/createUser/CreateUserForm";
+import { faUsers, faUserPlus, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+import styled from "styled-components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { UserRow } from "@/app/admin/adminActions";
+
+const PanelIcon = styled(FontAwesomeIcon)`
+    margin-right: 0.5em;
+`;
+
+interface Props {
+    userData: UserRow[];
+}
+
+// TODO VFB-23 Add accessibility tests for the admin page
+
+const AdminPage: React.FC<Props> = (props) => {
+    const adminPanels: [string, IconDefinition, ReactElement][] = [
```

Comment:
can you interface this and make it a list of objects? I'm not a massive fan of big lists that you only really know what's going on by their type and the usage (relying on the _user_ of the function to use clear variable names!)

Reply:
I've changed it now